discovery-infra/day2.py

```diff
@@ -96,16 +90,18 @@ def day2_nodes_flow(client,
                     cluster,
                     has_ipv_6,
                     num_worker_nodes,
-                    api_vip_ip,
-                    api_vip_dnsname,
                     install_cluster_flag,
                     day2_type_flag,
                     with_static_network_config,
                     base_cluster_name):
     tf_network_name, total_num_nodes = get_network_num_nodes_from_tf(tf_folder)
     with utils.file_lock_context():
         utils.run_command(
-            f'make _apply_terraform CLUSTER_NAME={terraform_cluster_dir_prefix}'
+            f'make _apply_terraform CLUSTER_NAME={terraform_cluster_dir_prefix} PLATFORM={consts.Platforms.BARE_METAL}'
+        )
+    with utils.file_lock_context():
+        utils.run_command(
+            f'make _apply_terraform CLUSTER_NAME={terraform_cluster_dir_prefix} PLATFORM={consts.Platforms.BARE_METAL}'
         )
```

Comment:
why do we do it twice?

Reply:
no exception is raised, so just added a loop